src/pages/programs/components/ProgramItem.tsx

```diff
@@ -45,7 +45,9 @@ const ProgramItem = ({ program }: { program: IPracticeProgram }) => {
               </div>
               <div className="flex flex-row items-center gap-x-1">
                 <ClockIcon className="w-4"></ClockIcon>
-                <p className="sm:text-sm lg:text-base text-xs">{`min. ${program.minWorkingHours}`}</p>
+                <p className="sm:text-sm lg:text-base text-xs">{`min. ${
```

Comment:
Momentan inchide PR. Vreau un task sa vedem cum putem face string interpolation cu libraria asta. M-am uitat in documentatie cateva minute dar nu am inteles exact. Merci.